### ISSUE TITLE
Unknown option AllowSupplementaryGroups

### ISSUE THREAD

**Issue by sfinke0** [edestecd/puppet-clamav#25]
Hi guys,

the option "AllowSupplementaryGroups" is not supported anymore since the ClamAV version 0.99.2+dfsg-0+deb8u1 on debian. (https://github.com/leapcode/leap_platform/pull/106)
Which says:

```
clamav (0.99.2+dfsg-0+deb8u1) stable; urgency=medium

* Import new Upstream.
* Drop AllowSupplementaryGroups option which is default now (Closes: #822444).
```

Could you fix that in your module please?

regards

**Comment by edestecd** [comment]
Yes, in the mean time you can use the workaround:

``` puppet
class { 'clamav':
  manage_freshclam  => true,
  freshclam_options => {
    'AllowSupplementaryGroups' => undef,
  },
}
```

**Comment by edestecd** [comment]
I'm about to run some builds on CentOS 7, which has the same version.
If it fails I'll yank this option from all...

**Comment by sheykh-sleky** [comment]
Have you had any progress in getting this removed?  Forgive my newbness...but where is that work around, um, worked around at?  I'm digging into it and hopefully I can figure it out.  Much appreciated!

Edit:  I believe I am closer; I've gotten this module loaded.  lol

**Comment by ubellavance** [comment]
Hi,
I don't know when it will be fixed as I don't have time to look at it and submit a PR so @edestecd would have to check when he has time.  The workaround is to set `'AllowSupplementaryGroups' => undef,` in your configuration. If you need more help, please show us your manifest(s).

**Comment by edestecd** [comment]
Solved via #60
